Readme.md

```diff
@@ -1,9 +1,8 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128641072/21.1.5%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E4288)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# Display Theme Selectors Based on BarItems and Hide Themes from List
+# WPF MVVM Behaviors - Display Theme Selectors Based on BarItems and Hide Themes from List
 
 DevExpress WPF Controls include the following behaviors that display a theme gallery in [Bars](https://docs.devexpress.com/WPF/DevExpress.Xpf.Bars.Bar) and [Ribbon](https://docs.devexpress.com/WPF/DevExpress.Xpf.Ribbon.RibbonControl):
```

Comment:
`in [Bars] and [Ribbon]` Bars and ribbon what? This sentence is incomplete

Reply:
in [Bars] and [Ribbon] controls